src/component/Board.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from "react";
+import Square from "./Square";
+import calculateWinner from "./caculationWinner";
+
+function Board() {
+  const [squares, setSquare] = useState(Array(9).fill(null));
+  const [xNext, setxNext] = useState(true);
+  const winner = calculateWinner(squares);
+  let status;
+
+  function HandleClick(i: number) {
```

Comment:
Should using arrow function. It will be better if in the future we need to use `React.useCallback` for this method